src/test/kotlin/no/nav/familie/klage/brev/BrevsignaturServiceTest.kt

```diff
@@ -2,22 +2,23 @@ package no.nav.familie.klage.brev
 
 import io.mockk.every
 import io.mockk.mockk
+import no.nav.familie.klage.infrastruktur.featuretoggle.FeatureToggleService
 import no.nav.familie.klage.personopplysninger.dto.Adressebeskyttelse
 import no.nav.familie.klage.personopplysninger.dto.PersonopplysningerDto
 import no.nav.familie.klage.testutil.BrukerContextUtil.testWithBrukerContext
+import no.nav.familie.kontrakter.felles.klage.Fagsystem
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 
 internal class BrevsignaturServiceTest {
-
-    val brevsignaturService = BrevsignaturService()
+    val brevsignaturService = BrevsignaturService(mockk<FeatureToggleService>(relaxed = true))
 
     @Test
     fun `skal anonymisere signatur hvis strengt fortrolig`() {
         val personopplysningerDto = mockk<PersonopplysningerDto>()
         every { personopplysningerDto.adressebeskyttelse } returns Adressebeskyttelse.STRENGT_FORTROLIG
 
-        val signaturMedEnhet = brevsignaturService.lagSignatur(personopplysningerDto)
+        val signaturMedEnhet = brevsignaturService.lagSignatur(personopplysningerDto, Fagsystem.EF)
```

Comment:
Her kunne det vært en idé å lage en `@ParameterizedTest` med `Fagsystem`, for å forsikre oss om at signatur blir anonymisert for alle fagsystemer

Reply:
Godt poeng, glemmer at det er en ting.